catalog/core/catalog-core-localstorageprovider/src/main/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProvider.java

```diff
@@ -369,12 +369,13 @@ private void commitUpdates(StorageRequest request) throws StorageException {
                     try {
                         Path createdTarget = Files.createDirectories(target);
                         List<Path> files = listPaths(contentIdDir);
-                        Files.copy(files.get(0),
-                                Paths.get(createdTarget.toAbsolutePath()
-                                                .toString(),
-                                        files.get(0)
-                                                .getFileName()
-                                                .toString()));
+                        for (Path path : files) {
+                            Path newTarget = Paths.get(createdTarget.toAbsolutePath().toString(), path.getFileName().toString());
```

Comment:
Is this line properly formatted? I would expect the second arg to get placed on its own line.